test/BetweenTest.php

```diff
@@ -18,37 +18,76 @@
  */
 class BetweenTest extends TestCase
 {
+    public function providerBasic()
+    {
+        return [
+            [
+                'min' => 1,
+                'max' => 100,
+                'inclusive' => true,
+                'expected' => true,
+                'values' => [1, 10, 100],
```

Comment:
If you're going to break this into a data provider, do a separate set per value as well.

Additionally, give each set a key with a descriptive name:

```php
'inclusive-valid-floor' => [
    'min' => 1,
    'max' => 100,
    'inclusive' => true,
    'expected' => true,
    'value' => 1,
],
'inclusive-valid-in-range' => [
    'min' => 1,
    'max' => 100,
    'inclusive' => true,
    'expected' => true,
    'value' => 1,
],
'inclusive-valid-ceiling' => [
    'min' => 1,
    'max' => 100,
    'inclusive' => true,
    'expected' => true,
    'value' => 100,
],
```

This makes it easy to identify which case to examine when a failure occurs, as PHPUnit will report the key; you can then search for that key to identify each set of data provided.